### ISSUE TITLE
NSP loss 수정

### ISSUE THREAD

**Issue by boykis82** [boykis82/TCL2021_telco_embedding#1]
`logits = tf.matmul(inp, W, transpose_b=True)      # (64*128*512)* (512,2)   -> (64*128*2)
print(logits.shape)
log_probs = tf.cast(tf.nn.log_softmax(logits, axis=-1), dtype=tf.float32)    # (64 * 128 * 2)
print(log_probs.shape)
#labels = tf.reshape(label, [-1])      # [64]
#print(labels.shape)
one_hot_labels = tf.one_hot(labels, depth=2, dtype=tf.float32)       #[64,2]
one_hot_labels = tf.reshape(one_hot_labels, (-1,1,2))
print(one_hot_labels.shape)

per_example_loss = -tf.reduce_sum(one_hot_labels * log_probs, axis=-1)
print(per_example_loss)

loss = tf.reduce_mean(per_example_loss)
loss`

**Comment by boykis82** [comment]
.
